cardano-cli/test/cardano-cli-golden/Test/Golden/Babbage/Governance/AnswerPoll.hs

```diff
@@ -30,43 +30,43 @@ hprop_golden_shelleyGovernanceAnswerPollNeg1Invalid = propertyOnce . H.moduleWor
 
   either (const H.success) (const H.failure) result
 
-hprop_golden_shelleyGovernanceAnswerPoll0 :: Property
-hprop_golden_shelleyGovernanceAnswerPoll0 = propertyOnce . H.moduleWorkspace "governance-answer-poll" $ \tempDir -> do
-  pollFile <- noteInputFile "test/cardano-cli-golden/files/golden/shelley/governance/polls/basic.json"
-  expectedAnswerFile <- noteInputFile "test/cardano-cli-golden/files/golden/shelley/governance/polls/basic.answer.0.json"
+hprop_golden_babbageGovernanceAnswerPoll0 :: Property
+hprop_golden_babbageGovernanceAnswerPoll0 = propertyOnce . H.moduleWorkspace "governance-answer-poll" $ \tempDir -> do
+  pollFile <- noteInputFile "test/cardano-cli-golden/files/golden/babbage/governance/polls/basic.json"
```

Comment:
This should be `test/cardano-cli-golden/files/input/governance/polls/basic.json`

That is:

* Replace `golden` with `input` because this is not a golden file.
* Delete `babbage`.